pysmt/cmd/installers/cvc4.py

```diff
@@ -1,92 +0,0 @@
-# Copyright 2014 Andrea Micheli and Marco Gario
```

Comment:
Do we have to remove this? 
This is a breaking change (that I think is ok) but then I'd probably combine this with a version bump.

Reply:
We could keep it if you prefer.